src/datanode/src/instance/grpc.rs

```diff
@@ -83,14 +87,14 @@ impl Instance {
         Ok(Output::AffectedRows(affected_rows))
     }
 
-    async fn handle_ddl(&self, request: DdlRequest) -> Result<Output> {
+    async fn handle_ddl(&self, request: DdlRequest, query_ctx: QueryContextRef) -> Result<Output> {
         let expr = request.expr.context(error::MissingRequiredFieldSnafu {
             name: "DdlRequest.expr",
         })?;
         match expr {
             DdlExpr::CreateTable(expr) => self.handle_create(expr).await,
             DdlExpr::Alter(expr) => self.handle_alter(expr).await,
-            DdlExpr::CreateDatabase(expr) => self.handle_create_database(expr).await,
+            DdlExpr::CreateDatabase(expr) => self.handle_create_database(expr, query_ctx).await,
             DdlExpr::DropTable(expr) => self.handle_drop_table(expr).await,
```

Comment:
By the way, is query_ctx needed for create/alter/drop operations?

Reply:
No, create/alter/drop table requests all carry catalog and schema.